ts/parsers/graphicFrameParser.ts

```diff
@@ -0,0 +1,74 @@
+//Graphic frame node includes tables, charts and diagrams
+
+import { getAttributeByPath } from "../helpers/attributesHandler";
+
+export default class GraphicFrameParser {
+  public static processGraphicFrameNodes = (graphicFrames) => {
+    const result = [];
+    for (const frame of graphicFrames) {
+      const graphicTypeUri = getAttributeByPath(
+        [frame],
+        ["a:graphic", "a:graphicData", "$"]
+      ).uri;
```

Comment:
could the helper be improved so URI can be also sent as the attribute required? e.g. ["a:graphic", "a:graphicData", "$", "uri"]

Reply:
yes, changed it. please check it out.